### ISSUE TITLE
Support DB_DSN for Craft 3.4+ Installs

### ISSUE THREAD

**Issue by jalendport** [burgerbuds/swiff#38]
In Craft 3.4, the ENV variables [changed](https://github.com/craftcms/cms/commit/4db33df006d93fcaf4bd2151d45d2af438f8c455) from using `DB_DRIVER`, `DB_SERVER`, `DB_DATABASE`, and `DB_PORT` to define your database config to just `DB_DSN`.

It would be great if Swiff could check for and use that value first and then fallback to searching for the old ENV variables (so older installs would continue to be supported).

**Comment by ben-rogerson** [comment]
Hey Jalen, sounds like a great improvement - I'm fairly busy with [twin](https://github.com/ben-rogerson/twin.macro) at the moment but I'll see what I can do.

**Comment by wihodges** [comment]
Yes any way to overide the .env database settings - would be nice. Just switched to DDEV docker so accessing DB is different location than where the app accesses it.
